cartographer/mapping/trajectory_node.h

```diff
@@ -67,8 +66,6 @@ struct TrajectoryNodes {
   std::vector<mapping::TrajectoryNode> trajectory_nodes;
 };
 
-proto::Trajectory ToProto(const std::vector<TrajectoryNode>& nodes);
```

Comment:
It will break when this PR gets merged, then @macmason will sent a PR to cartographer_ros to unbreak it. They will be merged very shortly after each other, so it should not be broken for long.